packages/extension/src/common/vscode/ext-types.ts

```
@@ -1019,6 +1019,27 @@ export class FileDecoration {
+}
+export namespace SyntaxTokenType {
+  type returnType = 'other' | 'comment' | 'string' | 'regex';
+
+  const tokenTypeMap: Record<SyntaxTokenType, returnType> = {
+    [SyntaxTokenType.Other]: 'other',
+    [SyntaxTokenType.Comment]: 'comment',
+    [SyntaxTokenType.String]: 'string',
+    [SyntaxTokenType.RegEx]: 'regex',
+  };
+
+  export function toString(v: SyntaxTokenType | unknown): returnType {
```

Biome

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)